CHANGES/898.feature.rst

```diff
@@ -0,0 +1 @@
+Add `without_query_keys` method.
```

Comment:
Plz use the past tense and explain what changed, with the end-users as the target audience in mind. This should be clear, informative and accurate. It's possible to write multiple sentences or paragraphs.

Remember to use RST and not Markdown (hence it's double backticks or even Sphinx's RST roles for referencing APIs).

Finally, you can insert a byline as shown in the examples here: https://github.com/aio-libs/yarl/tree/master/CHANGES#adding-change-notes-with-your-prs.